SeriesMaster-Frontend/App/Screens/DiscoverScreen.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import styled from "styled-components/native";
+import { StatusBar, View, Text } from "react-native";
+import Header from "../components/Header";
+import AppInputText from "../components/AppInputText";
+
+const Container = styled.View`
+  flex: 1;
```

Comment:
I don't see the corresponding parent element, which needs `display: flex` to make this line viable